pythonscript/embedded/godot/hazmat/ffi/script.py

```diff
@@ -12,6 +12,8 @@
 )
 from godot.bindings import Dictionary, Array
 
+# Set to True to show script loading progress
+verbose = True
```

Comment:
I would be better to provide this flag as a godot-controlled parameter (see https://github.com/touilleMan/godot-python/blob/66c9a6b42cefea70a13bd5be2f147dbdcfa480fb/pythonscript/embedded/godot/hazmat/ffi/init.py#L45)